manager/apis/app/v1alpha1/fybrikmodule_types.go

```diff
@@ -19,7 +19,7 @@ const (
 	// Workload indicates that the capability is available for all assets in the workload or is independent of assets
 	Workload CapabilityScope = "workload"
 
-	// Cluster indicates that a capability is available across workloads - i.e. across m4dapplication instances
+	// Cluster indicates that a capability is available across workloads - i.e. across Fybrikapplication instances
```

Comment:
Note that you are creating a conflict with the taxonomy work.
All these constant definitions are to be deleted and moved to taxonomy. 
See #804 
I suggest to document the cross reference of CapabilityScope between the various CR definitions.

Reply:
@ronenkat OK I'll open an issue for that.

Reply:
@ronenkat We are no longer checking `CapabilityScope` in the taxonomy so I don't think this will create a conflict